_pages/es/index.md

```diff
@@ -15,7 +15,7 @@ A campaign for Mozillians around the world to have impact in areas key to Mozill
 
 {: #area-deveng .area}
 [![image](/asserts/img/development.png)
-Empower developer communities]({{ "developer-engagement" | prepend: page.url | prepend: site.baseurl }})
+Empower developer communities](/es/developer-engagement/)
```

Comment:
I don't know Jenkyll that well, but shouldn't this handle the language on its own so that change would not be necessary?